posthog/api/test/test_property_definition.py

```diff
@@ -72,24 +74,24 @@ def test_pagination_of_property_definitions(self):
 
         response = self.client.get("/api/projects/@current/property_definitions/")
         self.assertEqual(response.status_code, status.HTTP_200_OK)
-        self.assertEqual(response.json()["count"], 308)
+        self.assertEqual(response.json()["count"], 311)
```

Comment:
This test suite is fairly annoying because it's based on the old demo data generation scheme – and demo data generators are NOT a good basis for tests, since they're pretty much random black boxes from a test's perspective. Nonetheless, I'm pretty certain this PR is actually an improvement, seeing that previously some properties (like `price`) were unaccounted for.